src/react-components/src/components/01-atoms/Button/Button.tsx

```diff
@@ -114,10 +114,11 @@ export default class Button extends React.Component<ButtonProps, {}> {
     }
 
     let btnCallback = mouseDown ? { onMouseDown: callback } : { onClick: callback };
+
     return (
       <button
         id={"btn-" + id}
-        className={bem(button_base_class, buttonModifiers, blockName)}
+        className={`${button_base_class} ${bem(button_base_class, buttonModifiers, blockName)}`}
```

Comment:
Is this our implementation of Bem being weird?  Bem won't apply the `button` class unless I explicitly use it.  Or should should all the button classes be @include-ing the button class

Reply:
Er, all paths lead to Rome, I guess

For something as atomic as buttons I think that they should probably have the button class applied, and in addition the option to base in blocks. I think what you've done here makes sense.